.github/workflows/theia-zowe-explorer-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Theia Zowe Explorer CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      theia:
+        image: theiaide/theia
```

Comment:
Since the image is failing every now and then, how about fixing its version to eliminate one source of potential randomness?

Reply:
I can do that.  I'm struggling with this a bit.  I'm try to start a theia service and connect to it from another container or from the VM.  I'm trying to see and learn more from the Jenkins pipeline for this.  If anyone has some time to take a look, please do!

Reply:
I have seen the Jenkins pipeline crash randomly for the same reason - that Theia did not start in time.